Business Rules/Dynamic Assignment Based on Business Hours/Script.js

```diff
@@ -0,0 +1,10 @@
+// Business Rule: Assign Incident Based on Business Hours
+// Trigger: Before Insert
+// Table: incident
+
+var currentHour = new GlideDateTime().getHourPart();
+if (currentHour >= 9 && currentHour <= 17) { // Business hours
```

Comment:
You would be better off determining whether a time is in a schedule than raw number comparison